src/libstd/primitive_docs.rs

```diff
@@ -362,8 +362,11 @@ mod prim_unit { }
 ///
 /// *[See also the `std::ptr` module](ptr/index.html).*
 ///
-/// Working with raw pointers in Rust is uncommon,
-/// typically limited to a few patterns.
+/// Working with raw pointers in Rust is uncommon, typically limited to a few patterns.
+/// Raw pointers can be unaligned or null when unused. However, when a raw pointer is used to
+/// load/store data from/to memory, they must be non-null and aligned.
+/// Storing through a raw pointer (`*ptr = data`) calls `drop` on the old value, so
```

Comment:
```suggestion
/// Storing through a raw pointer using `*ptr = data` calls `drop` on the old value, so
```
Maybe also say why calling drop here is problematic for uninit memory?

Reply:
Isn't that obvious? I can add "otherwise uninitialized data is dropped", but that seems redundant. Well, better to be clear I guess. I just don't want ti get overly verbose, that can also hinder readability.